sympy/functions/elementary/complexes.py

```diff
@@ -909,6 +1072,13 @@ class periodic_argument(Function):
     >>> periodic_argument(exp_polar(5*I*pi), pi)
     0
 
+    Parameters
+    ==========
+
+    arg : exp_polar, exp
+        Exp_polar class represents a polar number.
+        Exp represents an exponential expression.
```

Comment:
`eval` uses `ar, period` as literal names of arguments, so you should better change to those names to document.

But in my opinion, it is also incorrect that the first type should always be `exp_polar` and the second type is an instance of `Exp`. Both can just be general sympy `Expr, Expr` instance (For example, `periodic_argument(exp(I*pi), 3*pi)` just works and even first is actually neither `exp_polar` nor `exp` but `NegativeOne`)

Reply:
I actually documented only the first argument. I am changing the argument names now.

Reply:
What does Positive real number of infinity mean? In the explanation of principal_argument?